src/ucp/core/ucp_listener.c

```diff
@@ -369,6 +369,10 @@ void ucp_listener_destroy(ucp_listener_h listener)
     UCS_ASYNC_UNBLOCK(&listener->worker->async);
 
     ucp_listener_free_uct_listeners(listener);
+
+    if (listener->conn_reqs != 0) {
+        ucs_warn("pending %d connection requests", listener->conn_reqs);
```

Comment:
1. `ucs_warn("destroying listeneer %p with %u unprocessed connection requests"`
2. do it before line 371
3. space line after the "if" block

Reply:
Done